docker/Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM ubuntu:bionic
 
 ENV PYTHONIOENCODING UTF-8
 
+ARG DEBIAN_FRONTEND=noninteractive
```

Comment:
Debian frontend?

Reply:
It refers to the interface that is presented during the package installation.
Please see this for more info: https://askubuntu.com/a/972528

Reply:
In our case, it was prompting for user input and waiting on it during the installation. So the frontend used was `readline`. Needed to change that to above, to make it silent.

Reply:
Also, using the `ARG` command so that the environment variable is not persisted post image build.